src/tokenizer.ts

```diff
@@ -1537,8 +1534,7 @@ export class Tokenizer extends DiagnosticEmitter {
     throw new Error("not implemented"); // TBD
   }
 
-  readUnicodeEscape(): string {
-    var remain = 4;
+  private readSequenceEscape(remain: i32): string {
```

Comment:
Perhaps `readEscapeSequence`?

Reply:
Thanks, that's better

Reply:
Unfortunately this name already used

Reply:
Then perhaps just `readHexadecimalEscape`, which both old `readHexadecimalEscape` and `readUnicodeEscape` are, and remove the old ones - specifying `remain`.

Reply:
Done!